plugins/syntax/g_numeral.mlg

```diff
@@ -37,6 +37,8 @@ END
 VERNAC COMMAND EXTEND NumeralNotation CLASSIFIED AS SIDEFF
   | #[ locality = Attributes.locality; ] [ "Numeral" "Notation" reference(ty) reference(f) reference(g) ":"
       ident(sc) numnotoption(o) ] ->
-    { let (sigma, env) = Pfedit.get_current_context () in
+    { (* Global env is used here on purpose, in fact the type of vernac_string should not even take env/sigma *)
```

Comment:
What's `vernac_string`?

Reply:
> We can access the proof state (if it exists) and take the right env here, can't we?

I do believe that shouldn't be done in this case; maybe we should strengthen the underlying type.

 

> What's `vernac_string`?

Typo, should say `vernac_numeral_notation`. Ditto for g_newring, I do think the current type is misleading, these commands won't work correctly if anything other than the global env is used AFAICT so it is better just to remove the env sigma pair.